react/src/Participants/components/ParticipantTable.tsx

```diff
@@ -168,7 +175,10 @@ export default function ParticipantTable() {
             <MaterialTablePrimary
                 tableRef={tableRef}
                 columns={columns}
-                data={dataFetch}
+                data={query => {
+                    if (query) handleFilterChange(query.filters);
```

Comment:
if you can get away with firing this only at component tear-down, you could return the cache function from a useEffect hook so it fires just once. You'd have to grab and store the filters from `tableRef.current.state.query.filters`. Not sure it's necessary but it's an option.

Reply:
I'm not sure if I can grab the filters at tear-down. I ran into issues trying to do something similar in #577 because `tableRef.current` is null by that point.